projects/example-app/src/app/app.module.ts

```diff
@@ -43,7 +43,9 @@ import { AppRoutingModule } from '@example-app/app-routing.module';
     /**
      * @ngrx/router-store keeps router state up-to-date in the store.
      */
-    StoreRouterConnectingModule.forRoot(),
+    StoreRouterConnectingModule.forRoot({
+      onlyEventNavigationId: true,
```

Comment:
This won't solve the serializability checks, as the `routerState` is not serializable. 
I think we should introduce a `SerializableRouterStateSerializer` router in the example app to follow our own best practices on keeping actions and state serializable. We could even create this router in `@ngrx/router-store`?

Reply:
How about a `routerState` configuration flag that let's the user pick?

Full = DefaultRouterStateSerializer (what we have today),
Minimal = MinimalRouterStateSerializer (Basically this https://ngrx.io/guide/router-store/configuration#custom-router-state-serializer and the minimal route event info)

We could flip them at some point so the default is the fully serializable one.

Reply:
I like this idea! The only "catch" to it would be that it would be possible to say set the config to `Minimal` and provide a custom serializer (this would overrule the minimal config), which is a bit contradictory.

Reply:
True. I'd say we go with the minimal route event info either way, and override the custom one if they choose a `routerState`